fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/RepositoryObjectFactory.java

```diff
@@ -422,7 +422,7 @@ public void createMemberLink(URI parentUri, URI memberUri) throws FedoraExceptio
      * @param object
      */
     public void createRelationship(PID subject, Property property, Resource object) {
-        String sparqlUpdate = SparqlUpdateHelper.createSparqlInsert(subject.getRepositoryPath(), property, object);
+        String sparqlUpdate = SparqlUpdateHelper.createSparqlReplace(subject.getRepositoryPath(), property, object);
```

Comment:
I would actually suggest making another method named "createExclusiveRelationship", since other methods may need to support repeating relationships, and the other createRelationship method does not replace existing relations.

Reply:
good thought. i wasn't clear about all of the use-cases for this method